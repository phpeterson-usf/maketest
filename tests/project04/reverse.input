-h $(repo_dir)/reverse.hex $(TESTS_DIR)/reverse_s.s
