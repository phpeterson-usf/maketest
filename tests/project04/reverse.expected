v2.0 raw
e1a01000
e5d12000
e3520000
0a000001
e2811001
eafffffa
e2411001
e1500001
aa000006
e5d02000
e5d13000
e5c12000
e5c03000
e2800001
e2411001
eafffff6
e12fff1e
